test/model/tools/model_test/main.cpp

```diff
@@ -390,12 +390,22 @@ reduce_counterexample(std::shared_ptr<model::kv_workload> workload, const std::s
                 *w << op;
         }
 
+        /* Validate that we didn't produce a malformed workload. */
+        bool skip = false;
+        if (!w->verify_timestamps())
+            skip = true;
+
         /* Clean up the previous database directory, if it exists. */
```

Comment:
np: Looks like we assume it exists here.
```suggestion
        /* Clean up the previous database directory.*/
```

Reply:
If this is the first round of the algorithm, the home directory would not yet exist. `testutil_remove` does not return an error if the path does not exist, so no further check needs to be done for this call.